resources/views/posts/partials/form.edge

```diff
@@ -11,7 +11,7 @@
   @!component('components.label', { text: 'Title', related: 'title' })
   @!component('components.input', { name: 'title', value: old('title', post.title || ''), required: true })
   <div>
-    {{ elIf('<small class="text-xs">$self</small>', getErrorFor('title'), hasErrorFor('title')) }}
+    {{ elIf('<small class="form-text">$self</small>', getErrorFor('title'), hasErrorFor('title')) }}
```

Comment:
This is a Tailwind class so it would be better to change the test instead of the classname

Reply:
Then it needs to be changed here https://github.com/adonisjs/adonis-blog-demo/blob/ffee4eb50026014faeb41fafea4cb5ff6750e2a7/resources/views/posts/partials/form.edge#L23

Infact, the assertions must not be done on styling related classes, they are subject to change

Reply:
Yes, what about adding an html attribute `data-test-error` and use it to get the element?

This is used by Cypress.io and recommended when testing frontend.
https://docs.cypress.io/guides/references/best-practices.html#Selecting-Elements

Reply:
Yeah let’s do this